### PR TITLE
sending message to client via email

### PR DESCRIPTION
**Pull request by ilonalink**

### CODE REVIEW

File: cypress/page-objects/messagingPage.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        let clientValueToChooseFromDropDown = cy.get('a span strong').contains(clientName,{matchCase:false});
+        clientValueToChooseFromDropDown.click();
+    }
+
+    sendEmail(emailMessage: string) {
+        this.setTheMessage(emailMessage);
+        this.chooseToSendViaEmail();
+    }
+
+    setTheMessage(emailMessage: string) {
```
Comment (ShannyWorkiz) [inline cypress/page-objects/messagingPage.ts:35]: @ilonalink Make private

File: cypress/page-objects/messagingPage.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        cy.get('.textarea').type(emailMessage);
+    }
+
+    chooseToSendViaEmail() {
```
Comment (ShannyWorkiz) [inline cypress/page-objects/messagingPage.ts:39]: @ilonalink Make private

File: cypress/page-objects/messagingPage.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        cy.wait(5000);
+        SQLQueries.checkIfBodyMessageSent(email,message);
+    }
+
+    sendEmailToClient(clientName: string, emailMessage: string) {
+        this.openConversationWithClient(clientName);
+        this.sendEmail(emailMessage);
+    }
+
+    openConversationWithClient(clientName: string) {
```
Comment (ShannyWorkiz) [inline cypress/page-objects/messagingPage.ts:15]: @ilonalink Make private

File: cypress/page-objects/Clients/allClientsPage.ts [hunk 4]
```diff
@@ -3,9 +3,9 @@ import { CreateClientPage } from "./createClientPage";
 
 export class AllClientsPage {
 
-    createClient(): ClientPage {
+    createClient(email?: string): ClientPage {
```
Comment (IdoAdler2) [inline cypress/page-objects/Clients/allClientsPage.ts:6]: @ShannyWorkiz Perfect u did optional param. Thats the way to solve it :)

**Pull request merged**